lib/plugins/chatops.rb

```diff
@@ -15,12 +15,14 @@ def notify(message)
       case @service
       when 'slack'
         LOGGER.info("Sending Slack webhook message to #{@url}")
-        Chatops::Slack.new(@channel,
-                           @url,
-                           @user,
-                           message,
-                           http_options: @args[:http_options] || {},
-                           icon_emoji: @args[:icon_emoji]).notify
+        Chatops::Slack.new(
+          @channel, @url, @user, message, http_options: @args[:http_options] || {}, icon_emoji: @args[:icon_emoji]
+        ).notify
+      when 'rocketchat'
```

Comment:
Don't forget to `require 'plugins/chatops/rocketchat'`

Reply:
I knew I forgot something